src/main/resources/db/changelog/changes/03-insert-users.yaml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-users
+      author: Ruslan
+      changes:
+        - insert:
+            tableName: users
+            columns:
+              - column: {name: email, value: "bob@example.com"}
+              - column: {name: password, value: "1234"}
```

Comment:
passwords are stored encrypted in our application
don't forget to include plaintext password in a comment

Reply:
Yes, this will be in the next task.